docs/dev/lsp-extensions.md

```diff
@@ -333,7 +333,7 @@ Moreover, it would be cool if editors didn't need to implement even basic langua
 
 ### Unresolved Question
 
-* Should we return a nested brace structure, to allow paredit-like actions of jump *out* of the current brace pair?
+* Should we return a nested brace structure, to allow [paredit](https://paredit.org/)-like actions of jump *out* of the current brace pair?
```

Comment:
I didn't know what this was and couldn't figure it out. I believe that adding a link makes the document more helpful to a reader.